.github/workflows/ci-cd-main.yml

```
@@ -96,25 +143,55 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
-          cache: 'npm'
 
-      - name: 📦 Install Dependencies
+      - name: 📦 Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: ${{ env.PNPM_VERSION }}
+
+      - name: 💾 Get pnpm store directory
+        shell: bash
+        run: echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)